test/__snapshots__/test.js.snap

```diff
@@ -4558,6 +4558,129 @@ const Impl = require(\\"../implementations/URL.js\\");
 "
 `;
 
+exports[`URLCallback.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+module.exports = {
+  convert(value, { context = \\"The provided value\\" } = {}) {
+    if (typeof value !== \\"function\\" && !treatNonObjectAsNull) {
+      throw new TypeError(context + \\" is not a function\\");
+    }
+
+    function wrapper(...args) {
+      if (new.target !== undefined) {
+        throw new Error(
+          \\"Internal error: do not call converted callback function with \`new\`; instead use \\" +
+            \\"convertedFunc.construct()\\"
+        );
+      }
+
+      let returned;
+
+      for (let i = 0; i < args.length; i++) {
+        args[i] = utils.tryWrapperForImpl(args[i]);
+      }
+      if (args.length < 2) {
+        for (let i = args.length; i < 2; i++) {
+          args[i] = undefined;
+        }
+      } else if (args.length > 2) {
+        args.length = 2;
+      }
+
+      returned = Reflect.apply(value, this, args);
+
+      returned = convertURL(returned, { context: context });
```

Comment:
Can the generated code just use `{ context }`?

Reply:
We generate this object literal through a call to `Types.generateTypeConversion()`, which takes `"context"` as an argument. We could add a special case, but ideally this would be the job of Prettier, so I'm not too inclined to do that.